roles/openshift_node_dnsmasq/tasks/network-manager.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Install network manager dispatch script
+  copy:
+    src: networkmanager/99-origin-dns.sh
+    dest: /etc/NetworkManager/dispatcher.d/
+    mode: 0755
+  register: dispatch_script
+  
+- name: Restart Network Manager
+  service: 
+    name: NetworkManager
+    state: restarted
+  when: dispatch_script.changed | bool
+
+- name: Test dns resolution
+  command: >
+    getent hosts google.com
```

Comment:
Do we want to test against google.com? Won't this cause issues in disconnected environments?